apis/v1beta1/util/validation/common_test.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation_test
+
+import (
+	"testing"
+
+	gatewayv1b1 "sigs.k8s.io/gateway-api/apis/v1beta1"
+	validationtutils "sigs.k8s.io/gateway-api/apis/v1beta1/util/validation"
+)
+
+func TestContainsInSectionNameSlice(t *testing.T) {
```

Comment:
if we can reuse list generics then this test becomes unneeded.

Reply:
ACK

Reply:
https://github.com/kubernetes-sigs/gateway-api/pull/1601 removed this in https://github.com/kubernetes-sigs/gateway-api/commit/0c0e30a7c062a95afc51cb298d87cec930968328